### PR TITLE
Add service links to example and fix style

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Fixes: #1577
Also updated docs and header title param to conform with navigation links

**(BREAKING CHANGE)**
When using header service links the the text for the link is now set with `item.title` not `item.text` as before

### How to review
Example now looks as expected

### CODE REVIEW

**Comment by rmccar** [comment]
[Issue #1577: Service Links missing from internal header example and incorrectly styled when applied](https://trello.com/c/dgMpxFvF/1235-issue-1577-service-links-missing-from-internal-header-example-and-incorrectly-styled-when-applied)

File: src/components/header/examples/internal-basic/index.njk [hunk 1]
```diff
@@ -9,6 +9,22 @@
...
+                {
+                    "title": "Help",
+                    "url": "#0"
+                },
+                {
+                    "title": "My Account",
+                    "url": "#0"
+                },
+                {
+                    "title": "Styles",
```
Comment (jrbarnes9) [inline src/components/header/examples/internal-basic/index.njk:24]:
```suggestion
                    "title": "Sign out",
```

File: src/components/header/_header.scss [hunk 2]
```diff
@@ -134,6 +134,9 @@
     &__grid-top {
       min-height: 36px;
     }
+    &-service-nav__link {
+      color: $color-white;
+    }
```
Comment (jrbarnes9) [inline src/components/header/_header.scss:139]: I think the hover style for these links should match other white on colour links, same as the [promo banner](https://ons-design-system.netlify.app/components/promotional-banner/) and [announcement panel](https://ons-design-system.netlify.app/components/panel/examples/announcement/),  for example.

**Review by jrbarnes9 — CHANGES_REQUESTED** [review]
Can we also add `currentPath` to the census header examples?

**Review by boxadesign — APPROVED** [review]

**Review by jrbarnes9 — APPROVED** [review]

**Review by boxadesign — APPROVED** [review]

**Review by jrbarnes9 — APPROVED** [review]

**Pull request merged**